README.md

```diff
@@ -34,6 +34,10 @@ spec:
   image: ghcr.io/macstadium/orka-images/sonoma:latest
 ```
 
+### VM setup from IPSW script
+
+To get started with Orka using an IPSW and Orka Desktop, use the `orka-ipsw-setup.sh` script located in the `setup` directory.
```

Comment:
The script is just called `setup.sh` now, right?

Also - why not just link here directly to the script readme? (see [here](https://github.com/macstadium/orka-github-actions-integration/blob/41d3b6e69c75921c435ce3991f1a9abc7cff5c6b/README.md?plain=1#L47) for an example of what I mean by this)

Reply:
good catch, done, thanks!